docs/0___preamble.adoc

```diff
@@ -223,6 +239,7 @@ the first element of input argument `x` for function `setContinuousStates(..)` i
 
 If you plan to export or import models in FMI 2.0 format, you may find the following tools/models helpful for your development (available from https://fmi-standard.org/downloads):
 
+Referemce FMUs:: A set of hand-coded FMUs for development, testing and debugging of the Functional Mock-up Interface available at https://github.com/modelica/Reference-FMUs/
```

Comment:
Is this also true for the new FMI 2.0.4 features?

Reply:
Not yet to my knowledge, but it is the right place to look for examples